README.md

```diff
@@ -5,10 +5,60 @@
 </p>
 
 This repository contains some tools to be used by forensics teams to collect
-evidence from cloud platforms. Currently, Google Cloud Platform and Amazon Web Services are supported.
+evidence from cloud platforms. Currently, Google Cloud Platform, Microsoft Azure,
+and Amazon Web Services are supported.
+
+It consists in one module called `libcloudforensics` which implements functions
+that can be desirable in the context of incident response in a cloud
+environment, as well as a CLI wrapper tool for these functions.
 
 ## Quick install
 
 ```
 pip install libcloudforensics
 ```
+
+## Running the CLI tool
+
+A standalone tool called `cloudforensics` is created during installation.
+
+```
+$ cloudforensics --help
+usage: cloudforensics [-h] {aws,az,gcp} ...
+
+CLI tool for AWS and GCP.
```

Comment:
I added Azure to the cli.py file as well. This should be:
CLI tools for AWS, Azure and GCP

Reply:
Fixed

Reply:
Don't we still need to update this text manually and add Azure here?

Reply:
indeed :)